merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinBindings.java

```diff
@@ -221,7 +225,7 @@ private void getConstraintViolations(final Context ctx) {
       final var planId = input.planId();
       final var simulationDatasetId = input.simulationDatasetId();
 
-      final var constraintViolations = this.simulationAction.getViolations(planId, simulationDatasetId);
```

Comment:
`getViolations` needs to be removed from `GetSimulationResultsAction`

Reply:
Forgot to do this as a cleanup step!